lib/cast/cast_sender.js

```diff
@@ -192,21 +195,9 @@ shaka.cast.CastSender = class {
       // If the API becomes available before this instance dies, init will be
       // called again.
 
-      // A note about this value: In our testing environment, we load both
-      // uncompiled and compiled code.  This global callback in uncompiled mode
-      // can be overwritten by the same in compiled mode.  The two versions will
-      // each have their own instances_ map.  Therefore the callback must have a
-      // name, as opposed to being anonymous.  This way, the CastSender tests
-      // can invoke the named static method instead of using a global that could
-      // be overwritten.
-      if (!window.__onGCastApiAvailable) {
-        window.__onGCastApiAvailable = shaka.cast.CastSender.onSdkLoaded_;
-      }
-      if (window.__onGCastApiAvailable != shaka.cast.CastSender.onSdkLoaded_) {
-        shaka.log.alwaysWarn('A global Cast SDK hook is already installed! ' +
-          'Shaka Player will be unable to receive a notification when the ' +
-          'Cast SDK is ready.');
-      }
+      // save pre-existing __onGCastApiAvailable in order to restore later
+      this.__onGCastApiAvailable_ = window.__onGCastApiAvailable || null;
+      window.__onGCastApiAvailable = (av) => this.onGCastApiAvailable_(av);
```

Comment:
Nit: please call this "available" instead of "av".  I completely forgot what that variable was or did from looking at the shortened name.

Reply:
Sure! Done.